### PR TITLE
Fix build

### PR DESCRIPTION
**Pull request by paususe**
Put SUSEconnect command in preformatted boundaries and add missing ifeval for Uyuni

**Please make sure you always build (`make antora-suma`/ `make antora-uyuni`) before pushing anything to the repo, or even before committing. When I try to build the docs, quite often they just do not build.**

### CODE REVIEW

File: modules/quickstart-public-cloud/pages/installation.adoc [hunk 1]
```diff
@@ -8,7 +8,9 @@ For detailed instructions on registering {productname} to {scc}, see xref:instal
 When you have registered, all {sle} modules will be activated.
 You will also need to activate the Public Cloud module:
 
+```
 # SUSEConnect -p sle-module-public-cloud/15.1/x86_64
+```
```
Comment (Loquacity) [inline modules/quickstart-public-cloud/pages/installation.adoc]:
```suggestion
----
SUSEConnect -p sle-module-public-cloud/15.1/x86_64
----
```

File: modules/quickstart-public-cloud/pages/installation.adoc [hunk 2]
```diff
@@ -35,7 +37,7 @@ When you have logged in to the administrator account for the first time, change
 Add the required software products to your client, and schedule a repository synchronization.
 
 
-
+ifeval::[{uyuni-content} == true]
```
Comment (Loquacity) [inline modules/quickstart-public-cloud/pages/installation.adoc]: This isn't required, the endif needs to be deleted instead. It's just copypasta. I thought I took it out already, sorry.

**Comment by jcayouette** [comment]
Using superpowers to merge since this is affecting our build.

**Comment by Loquacity** [comment]
> Using superpowers to merge since this is affecting our build.

We shouldn't be publishing the QS at all ...

**Pull request merged**